lib/src/config/login_options.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter_login/src/service/validation.dart';
+
+class LoginOptions {
+  const LoginOptions({
+    this.image,
+    this.title,
+    this.subtitle,
+    this.emailLabel,
+    this.passwordLabel,
+    this.emailInputPrefix,
+    this.passwordInputPrefix,
+    this.decoration = const InputDecoration(),
+    this.initialEmail = '',
+    this.initialPassword = '',
+    this.translations = const LoginTranslations(),
+    this.validationService,
+    this.loginButtonBuilder = _createLoginButton,
+    this.forgotPasswordButtonBuilder = _createForgotPasswordButton,
+    this.requestForgotPasswordButtonBuilder =
+        _createRequestForgotPasswordButton,
+    this.registrationButtonBuilder = _createRegisterButton,
+    this.emailInputContainerBuilder = _createEmailInputContainer,
+    this.passwordInputContainerBuilder = _createPasswordInputContainer,
+  });
+
+  final ButtonBuilder loginButtonBuilder;
+  final ButtonBuilder registrationButtonBuilder;
+  final ButtonBuilder forgotPasswordButtonBuilder;
+  final ButtonBuilder requestForgotPasswordButtonBuilder;
+  final InputContainerBuilder emailInputContainerBuilder;
+  final InputContainerBuilder passwordInputContainerBuilder;
+
+  final Widget? image;
+  final Widget? title;
+  final Widget? subtitle;
+  final Widget? emailLabel;
+  final Widget? passwordLabel;
+  final Widget? emailInputPrefix;
+  final Widget? passwordInputPrefix;
+  final InputDecoration decoration;
+  final String initialEmail;
+  final String initialPassword;
+  final LoginTranslations translations;
+  final ValidationService? validationService;
+
+  ValidationService get validations =>
+      validationService ?? LoginValidationService(this);
+}
+
+class LoginTranslations {
+  const LoginTranslations({
+    this.emailEmpty = 'Email is required',
+    this.passwordEmpty = 'Password is required',
+    this.emailInvalid = 'Enter a valid email address',
+  });
+
+  final String emailInvalid;
+  final String emailEmpty;
+  final String passwordEmpty;
+}
+
+Widget _createEmailInputContainer(Widget child) => child;
+
+Widget _createPasswordInputContainer(Widget child) => child;
+
+Widget _createLoginButton(
+  BuildContext context,
+  OptionalAsyncCallback onPressed,
+  bool disabled,
+) {
+  return Opacity(
+    opacity: disabled ? 0.5 : 1.0,
+    child: ElevatedButton(
+      onPressed: onPressed,
+      child: const Text('Login'),
+    ),
+  );
+}
+
+Widget _createForgotPasswordButton(
+  BuildContext context,
+  OptionalAsyncCallback onPressed,
+  bool disabled,
+) {
+  return Opacity(
+    opacity: disabled ? 0.5 : 1.0,
+    child: TextButton(
+      onPressed: onPressed,
+      child: const Text('Forgot password?'),
+    ),
+  );
+}
+
+Widget _createRequestForgotPasswordButton(
+  BuildContext context,
+  OptionalAsyncCallback onPressed,
+  bool disabled,
+) {
+  return Opacity(
+    opacity: disabled ? 0.5 : 1.0,
+    child: TextButton(
+      onPressed: onPressed,
+      child: const Text('Forgot password?'),
```

Comment:
```suggestion
      child: const Text('Reset password'),
```

Reply:
I think this is the submit button so maybe 'Reset password' or 'Send reset email' is better here?